src/test/java/HomePageTest.java

```diff
@@ -7,26 +8,24 @@
 public class HomePageTest extends BaseTest{
 
     @Test
-    public void SignUpSuccess () throws InterruptedException {
-
-        driver.get("https://kwidos.com/auth/register/contractor");
+    public void SignUpSuccess () {
         List<Character> charSet = new ArrayList<>(Arrays.asList('a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', '1', '2'));
         Collections.shuffle(charSet, new Random());
         String rand="";
         for (int i = 0; i < charSet.size(); i++) {
             rand=rand+charSet.get(i);
         }
-        Thread.sleep(4000);
+        driver.get("https://kwidos.com/auth/register/contractor");
+        wait.until(ExpectedConditions.presenceOfElementLocated(By.cssSelector("[formcontrolname='firstName']")));
         driver.findElement(By.cssSelector("[formcontrolname='firstName']")).sendKeys("George");
         driver.findElement(By.cssSelector("[formcontrolname='lastName']")).sendKeys("Suz");
```

Comment:
you use the same locators a few times. It would be better to store locator in separate private static final variable and reuse it all tests